go/internal/feast/onlinestore/cassandraonlinestore.go

```diff
@@ -0,0 +1,299 @@
+package onlinestore
+
+import (
+	"context"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/feast-dev/feast/go/internal/feast/registry"
+	"github.com/feast-dev/feast/go/internal/feast/utils"
+	"github.com/feast-dev/feast/go/protos/feast/serving"
+	"github.com/feast-dev/feast/go/protos/feast/types"
+	"github.com/gocql/gocql"
+	"github.com/golang/protobuf/proto"
+	"github.com/rs/zerolog/log"
+
+	"google.golang.org/protobuf/types/known/timestamppb"
+	gocqltrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/gocql/gocql"
+)
+
+type CassandraOnlineStore struct {
+	project string
+
+	// Cluster configurations for Cassandra/ScyllaDB
+	clusterConfigs *gocqltrace.ClusterConfig
+
+	// Session object that holds information about the connection to the cluster
+	session *gocqltrace.Session
+
+	// keyspace of the table. Defaulted to using the project name
```

Comment:
if not specified, set to scylladb. It's set to `feast_keyspace` in CassandraOnlineStoreConfig in python. Bring the consistency to the keyspace name.

Reply:
Done